### DISCUSSION TITLE
Error when developing Laravel Package (via spatie/package-skeleton-laravel) Error: ENAMETOOLONG: name too long

### DISCUSSION THREAD

**Discussion by nathangross** [orchestral/testbench#390] [Q&A] [answered]
I'm developing a package for laravel and using Spatie's repo to help out. Somewhere along the way I started to get a bug I'm not exactly sure what is causing it. I [posted](https://github.com/spatie/package-skeleton-laravel/discussions/321) about this in their repo discussions but thought it might be relevant here too.

Everything was working great, but then at some point I started to notice my Tailwind InteliSense (VS Code plugin) was not working right. I looked at the output tab for that plugin and noticed this error:

```
Error: ENAMETOOLONG: name too long, open '/Users/nathan/Code/voa-festival-splash/vendor/building13/laravel-blackbird-ui/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/orchestra/testbench-core/laravel/vendor/symfony/error-handler/Resources/assets/css/exception_full.css'
```
It's probably clear but for context: I'm working locally on my package: building13/laravel-blackbird-ui/ via a local laravel project: voa-festival-splash

It seems like some weird loop is happening somewhere but I could not figure out where. So in the meantime I've just removed the Orchestra package and things seem ok for now.

A few things I've tried while troubleshooting:

- installed my package in another project = same error
- installed another fresh package built from this skeleton = same error

Any ideas what might be going on here?

**Comment by crynobone** [comment, marked as answer]
You should exclude testbench-core dorectories in VS Code plug-in: https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss

**Reply by nathangross** [reply]
That was simple and that seems to work. Thank you for the reply!
